AMW_angular/io/src/app/resources/resource-edit/resource-edit.component.html

```diff
@@ -0,0 +1,8 @@
+<app-loading-indicator [isLoading]="isLoading()"></app-loading-indicator>
+<app-page>
+  <div class="page-title">Edit {{ this.resource()?.name }}</div>
+
+  <div class="page-content">
+    <div class="container">Loaded resource: {{ this.resource()?.name }}</div>
```

Comment:
Is it necessary to repeat resource name here too, if you are already defining it at the page-title level?

Reply:
it's just some dummy content